src/plugins/kibana/public/settings/sections/indices/partials/_edit_field_type.html

```diff
@@ -0,0 +1,13 @@
+<select ng-if="knownFieldTypes[field.name] !== 'geo_point'" name="field_type" ng-model="field.type">
```

Comment:
Not 100% sure I understand the question, are you asking if a geo_point field could be of type string in the elasticsearch mappings?